applications/ConstitutiveLawsApplication/custom_constitutive/associative_plastic_damage_model.h

```diff
@@ -99,15 +99,15 @@ class KRATOS_API(CONSTITUTIVE_LAWS_APPLICATION) AssociativePlasticDamageModel
     KRATOS_CLASS_POINTER_DEFINITION(AssociativePlasticDamageModel);
 
     struct PlasticDamageParameters {
-        BoundedMatrixType ComplianceMatrixIncrement = ZeroMatrix(VoigtSize,VoigtSize);
-        BoundedMatrixType ComplianceMatrix       = ZeroMatrix(VoigtSize,VoigtSize);
-        BoundedMatrixType ConstitutiveMatrix     = ZeroMatrix(VoigtSize,VoigtSize);
-        BoundedMatrixType TangentTensor          = ZeroMatrix(VoigtSize,VoigtSize);
-        BoundedVectorType PlasticFlow            = ZeroVector(VoigtSize);
-        BoundedVectorType PlasticStrain          = ZeroVector(VoigtSize);
-        BoundedVectorType PlasticStrainIncrement = ZeroVector(VoigtSize);
-        BoundedVectorType StrainVector           = ZeroVector(VoigtSize);
-        BoundedVectorType StressVector           = ZeroVector(VoigtSize);
+        BoundedMatrixType ComplianceMatrixIncrement { ZeroMatrix(VoigtSize,VoigtSize) } ;
```

Comment:
Interesting... Why is it needed??

Reply:
honestly I dont know, @pooyan-dadvand ?

Reply:
Here you are explicitly calling the constructor and seems that for some reason GCC 4.8 has less problem in finding the correct constructor.